docs/runtime/compiler-ir.md

```diff
@@ -12,12 +12,18 @@ for future goals.
 ## Visualization
 
```

Comment:
We should stress, that the stability of this API is [private only](https://netbeans.apache.org/wiki/main/wiki/API_Stability/#private) and can be changed/removed at any moment.